examples/deployment/docker/db_client/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.8
+
+RUN apt-get update && \
+    apt-get install -y mysql-client
+
+ADD . /go/src/github.com/google/trillian
+WORKDIR /go/src/github.com/google/trillian
+
+RUN go get -v ./server/trillian_log_server
+
+ENV DB_USER=test \
+    DB_PASSWORD=zaphod \
+    DB_DATABASE=test
+
+RUN ./examples/deployment/scripts/download-wait-for-it.sh
```

Comment:
in the `[docker-compose.yaml](https://github.com/google/trillian/pull/667/files#diff-971a49e212add79c1dc8922a40d5e753R14)` file, this is used to wait for MySQL to become available before attempting to run `resetdb.sh`